lib/types/themeLibrary/themeLibraryType.js

```diff
@@ -0,0 +1,15 @@
+const ThemeLibraryFormatter = require("./ThemeLibraryFormatter");
+const ThemeLibraryBuilder = require("./ThemeLibraryBuilder");
+
+module.exports = {
+	format: function(project) {
+		return new ThemeLibraryFormatter({project}).format();
+	},
+	build: function({resourceCollections, tasks, project, parentLogger, buildContext}) {
+		return new ThemeLibraryBuilder({resourceCollections, project, parentLogger, buildContext}).build(tasks);
+	},
+
+	// Export type classes for extensibility
+	Builder: ThemeLibraryBuilder,
```

Comment:
does "for extensibility" only mean that consumers can extend those classes or does it also mean that they can inject own implementations here via these two names? If so, the new expressions above should use the properties of the export, not the import names.

Reply:
Yes, subclassing